.thoth.yaml

```diff
@@ -0,0 +1,29 @@
+
+host: khemenu.thoth-station.ninja
+tls_verify: false
+requirements_format: pipenv
+
+runtime_environments:
+  - name: 'rhel:8'
+    operating_system:
+      name: rhel
+      version: "8.0"
+    python_version: "3.8"
```

Comment:
this project is using python 3.6 https://github.com/thoth-station/Github-Issues-Classifier/blob/db3f1def3756f03de137814cc2d793394d43fe58/Pipfile#L25

Reply:
Updated to py38 now :)